_posts/2013-04-11-testing.md

```diff
@@ -60,20 +60,20 @@ import { test } from 'ember-qunit';
 import startApp from '../helpers/start-app';
 var App;
 
-module('An Integration test', {
-  setup: function() {
+Qunit.module('An Integration test', {
```

Comment:
It will be with QUnit 2.0 (http://qunitjs.com/upgrade-guide-2.x/). The blueprints now generate tests like this (#3197 was merged a few days ago)